partiql-ast/src/ast.rs

```diff
@@ -470,13 +470,13 @@ pub struct Path {
 #[derive(Clone, Debug, PartialEq)]
 pub struct Call {
     pub func_name: SymbolPrimitive,
-    pub args: Vec<Box<Expr>>,
+    pub args: Option<Vec<Box<Expr>>>,
```

Comment:
Is having an `Option` here actually adding anything over having an empty `Vec` ?

Reply:
At first went with empty Vec but then thought using Option might make more sense in signaling that we can have functions with no args. Happy to change though, do you see a performance or ergonomics concern?

Reply:
I think a vec with no args is preferable. Otherwise we have 2 situations that represent 0 args: `None` and `Some(vec![])`

Reply:
That's a fair point, will make the change.